websites/R/RhysBOT/presence.ts

```diff
@@ -0,0 +1,41 @@
+const presence = new Presence({
+	clientId: "658765364439810048"
+});
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "https://i.imgur.com/yAhhPzj.png",
```

Comment:
I assume this service is owned by you, if so, this asset should be uploaded to Discord Application Portal